R/scripts/build.sh

```diff
@@ -98,6 +142,8 @@ fi
 
 export PKG_LIBS="${C_LIBS_FLAGS//_EXEC_ROOT_/${EXEC_ROOT}/}"
 export PKG_CPPFLAGS="${C_CPP_FLAGS//_EXEC_ROOT_/${EXEC_ROOT}/}"
+export PKG_FCFLAGS="${PKG_CPPFLAGS}"  # Fortran 90/95
```

Comment:
If I remember correctly, yes it is only about --coverage and -O0, and, e.g., the includes are not taken into account. I did not include any fortran code because of an incompatible for code coverage between gfortran and clang, and anyway I thought of this for exhaustivity, but I don't maintain any Fortran code.  I gather you don't either, so I think this line and the following can probably be removed.